### PR TITLE
Add link to repositories page

### PR DESCRIPTION
**Pull request by kjlundsgaard**
Hides files changed list, commit author stats, and other projects affected if the organization does not have any repositories associated. 

Currently looks like:
screen shot 2017-03-10 at 3 16 04 pm

### CODE REVIEW

File: src/sentry/static/sentry/less/releases.less [hunk 1]
```diff
@@ -85,3 +85,12 @@
   background-color: @green;
   margin-top: 9px;
 }
+
+/**
+* Connect Repository
+* ============================================================================
+*/
+
+.connect-repository {
```
Comment (ckj) [inline src/sentry/static/sentry/less/releases.less]: We actually don't need this bit as we have an `.align-center` utility class. I'm working on this section rn and will apply the fix.

Comment (kjlundsgaard) [inline src/sentry/static/sentry/less/releases.less]: 👍

**Review by ckj — COMMENTED** [review]
Looks good design-wise!

File: src/sentry/static/sentry/app/views/releases/releaseOverview.jsx [hunk 2]
```diff
@@ -163,48 +184,66 @@ const ReleaseOverview = React.createClass({
...
-              {numCollapsed > 0 && <Collapsed onClick={this.onCollapseToggle} count={numCollapsed}/>}
-            </ul>
+            {hasRepos &&
+              <div>
+                <h5>{fileCount} Files Changed</h5>
+                <ul className="list-group list-group-striped m-b-2">
+                  {files.map(filename => {
+                    return (
+                      <FileChange
+                        key={fileChangeSummary[filename].id}
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/releases/releaseOverview.jsx]: Nitpick, but I probably would have aliased these above:

```javascript
let {id, authors, types} = fileChangeSummary[filename];
```

**Review by benvinegar — APPROVED** [review]
This might suffer from some UI pop-in while we wait for data to load (w/ no loading graphic) ... but hey, let's try it.

**Pull request merged**